submissions/ses3332002/html-css-popup/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="build\style.css">
```

Comment:
It seems the path to your styles can broke here
```suggestion
  <link rel="stylesheet" href="build/style.css">
```

Reply:
Well, I practice on Linux at home and on Windows at work :) 
Maybe it remained from Windows session.